src/bin/dual-iir.rs

```diff
@@ -142,6 +147,15 @@ pub struct Settings {
     /// # Value
     /// See [signal_generator::BasicConfig#miniconf]
     signal_generator: [signal_generator::BasicConfig; 2],
+
+    /// Pounder output control
+    ///
+    /// # Path
+    /// `pdh`
+    ///
+    /// # Value
+    /// See [PDHLockGeneratorConfig#miniconf]
+    pdh: PDHLockGeneratorConfig,
```

Comment:
This should be wrapped in an `Option`. It should only be `Some(pdh)` if pounder is found to be present. Miniconf will transparently handle the `Some` or `None` variants as "existing" or "not existing" at run time.

Reply:
That way we can support for `lockin` with and without pounder present.